coreos-with-fleet-multivm/cloud-config.yaml

```diff
@@ -0,0 +1,14 @@
+#cloud-config
+
+coreos:
+  etcd:
+    # generate a new token for each unique cluster from https://discovery.etcd.io/new
+    discovery: https://discovery.etcd.io/7a47ef32f1f05e2ce9eb3e4782e04e0b
+    # deployments across multiple cloud services will need to use $public_ipv4
```

Comment:
It would be nice not to have to provide this but I'm not sure how to achieve this in the context of an ARM template.